tests/x11/gnome_control_center.pm

```diff
@@ -15,9 +15,12 @@
 use base "x11test";
 use strict;
 use testapi;
-
+use utils 'ensure_unlocked_desktop';
 
 sub run {
+    # If system update tests were executed, need to switch back to x11
+    select_console('x11', await_console => 0);
+    ensure_unlocked_desktop;
```

Comment:
This looks like it will not scale well but ok for now

Reply:
That's what I've mentioned in the ticket. Let's see if it will break anything.